### DISCUSSION TITLE
Do Too few `Median fraction of fragments overlapping peaks` lead to few features?

### DISCUSSION THREAD

**Discussion by jiangpuxuan** [GreenleafLab/ArchR#1611] [Questions / Documentation]
The `Median fraction of fragments overlapping peaks` of my dataset is 1.421%. When I ran to `getMarkerFeatures()` or `getMarkers()`, the cell clusters in my dataset turned to have too few markers or marker peaks..... ( only 7 markerpeaks in all 4 cell clusters, FDR<0.05, log2FC>1)
I would like to figure out whether it is caused by my poor `Median fraction of fragments overlapping peaks` .
Thank you for your help!

Here is my archR project:
```
class: ArchRProject 
outputDirectory: /Array-Tuqiang/pxjiang/analysis/atac/CRC_P07A/proj_crc07_hg19 
samples(1): Human_0721
sampleColData names(1): ArrowFiles
cellColData names(34): Sample TSSEnrichment ... ReadsInPeaks FRIP
numberOfCells(1): 1364
medianTSS(1): 6.142
medianFrags(1): 3717
```

**Comment by rcorces** [comment]
Something does not seem right here. The fraction of fragments overlapping peaks is far too low.

**Reply by jiangpuxuan** [reply]
> Something does not seem right here. The fraction of fragments overlapping peaks is far too low.

Sorry, does it mean that my dataset cannot be used in downstream analysis?

**Comment by huangmiao1996** [comment]
Hey, I also encountered the question. Could we talk together for further investigation in Wechat? If so, you could contact with me by email him1996@163.com.
